api/models/version.go

```diff
@@ -101,11 +101,11 @@ func (v *Version) Patch(patch *VersionPatch) error {
 	return nil
 }
 
-func (v *Version) BeforeCreate(scope *gorm.Scope) {
+func (v *Version) BeforeCreate(db *gorm.DB) error {
```

Comment:
I just checked the doc and found that gorm v2 updates their hooks: https://gorm.io/docs/create.html#Create-Hooks